2021-09-17 22:19:55,959 INFO    MainThread:8640 [wandb_setup.py:_flush():69] setting env: {}
2021-09-17 22:19:55,959 INFO    MainThread:8640 [wandb_setup.py:_flush():69] setting login settings: {}
2021-09-17 22:19:55,959 INFO    MainThread:8640 [wandb_init.py:_log_setup():337] Logging user logs to /Users/luisfelipevillaarenas/Documents/T-con/Projects/predictive_care/cpe-replacement/wandb/run-20210917_221955-1tqh29hk/logs/debug.log
2021-09-17 22:19:55,959 INFO    MainThread:8640 [wandb_init.py:_log_setup():338] Logging internal logs to /Users/luisfelipevillaarenas/Documents/T-con/Projects/predictive_care/cpe-replacement/wandb/run-20210917_221955-1tqh29hk/logs/debug-internal.log
2021-09-17 22:19:55,960 INFO    MainThread:8640 [wandb_init.py:_jupyter_setup():288] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f9b26e94040>
2021-09-17 22:19:55,961 INFO    MainThread:8640 [wandb_init.py:init():370] calling init triggers
2021-09-17 22:19:55,961 INFO    MainThread:8640 [wandb_init.py:init():375] wandb.init called with sweep_config: {}
config: {}
2021-09-17 22:19:55,961 INFO    MainThread:8640 [wandb_init.py:init():419] starting backend
2021-09-17 22:19:55,961 INFO    MainThread:8640 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2021-09-17 22:19:55,979 INFO    MainThread:8640 [backend.py:ensure_launched():135] starting backend process...
2021-09-17 22:19:55,996 INFO    MainThread:8640 [backend.py:ensure_launched():139] started backend process with pid: 8668
2021-09-17 22:19:56,000 INFO    MainThread:8640 [wandb_init.py:init():424] backend started and connected
2021-09-17 22:19:56,004 INFO    MainThread:8640 [wandb_run.py:_label_probe_notebook():845] probe notebook
2021-09-17 22:19:56,005 INFO    MainThread:8640 [wandb_run.py:_label_probe_notebook():852] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-09-17 22:19:56,006 INFO    MainThread:8640 [wandb_init.py:init():472] updated telemetry
2021-09-17 22:19:56,006 INFO    MainThread:8640 [wandb_init.py:init():491] communicating current version
2021-09-17 22:19:57,100 INFO    MainThread:8640 [wandb_init.py:init():496] got version response upgrade_message: "wandb version 0.12.2 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-09-17 22:19:57,103 INFO    MainThread:8640 [wandb_init.py:init():504] communicating run to backend with 30 second timeout
2021-09-17 22:19:57,697 INFO    MainThread:8640 [wandb_init.py:init():529] starting run threads in backend
2021-09-17 22:20:02,113 INFO    MainThread:8640 [wandb_run.py:_console_start():1624] atexit reg
2021-09-17 22:20:02,115 INFO    MainThread:8640 [wandb_run.py:_redirect():1498] redirect: SettingsConsole.WRAP
2021-09-17 22:20:02,116 INFO    MainThread:8640 [wandb_run.py:_redirect():1535] Wrapping output streams.
2021-09-17 22:20:02,121 INFO    MainThread:8640 [wandb_run.py:_redirect():1559] Redirects installed.
2021-09-17 22:20:02,121 INFO    MainThread:8640 [wandb_init.py:init():554] run started, returning control to user process
2021-09-17 22:20:02,122 INFO    MainThread:8640 [wandb_run.py:_config_callback():873] config_cb None None {'lr': 0.001, 'dropout_prob': 0.1, 'weight_decay': 0.05, 'latent_dim': 500, 'inter_dim': 32, 'output_dim': 1, 'freeze_encoder': True, 'encoder_thermal': None, 'encoder_base': None, 'name_labels_logger': ['label_minor_physical_thermal_damage']}
2021-09-17 22:30:08,117 INFO    MainThread:8640 [wandb_init.py:_pause_backend():250] pausing backend
2021-09-17 22:30:08,117 INFO    MainThread:8640 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-09-17 22:31:05,282 INFO    MainThread:8640 [wandb_init.py:_resume_backend():259] resuming backend
2021-09-17 22:31:05,352 INFO    MainThread:8640 [wandb_init.py:_pause_backend():250] pausing backend
2021-09-17 22:31:05,352 INFO    MainThread:8640 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-09-17 22:31:11,753 INFO    MainThread:8640 [wandb_run.py:_atexit_cleanup():1594] got exitcode: 0
2021-09-17 22:31:11,754 INFO    MainThread:8640 [wandb_run.py:_restore():1566] restore
2021-09-17 22:31:12,012 INFO    MainThread:8640 [wandb_run.py:_restore():1566] restore
